Bottom: 7556dbf7e749e854ef7ae66a8aaf42140e1f08d4
Top:    b2a2f9c3fc752c3fcbf383cddc91b01bdd3f9b6f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-23 10:38:29 +0000

Suggested fix to Select(NetByName,...)


---

diff --git a/src/select.c b/src/select.c
index c28da80..ce18b64 100644
--- a/src/select.c
+++ b/src/select.c
@@ -1017,7 +1017,6 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
       InitConnectionLookup ();
       ResetFoundPinsViasAndPads (false);
       ResetFoundLinesAndPolygons (false);
-      SaveUndoSerialNumber ();
 
       MENU_LOOP (&PCB->NetlistLib);
       {
@@ -1035,8 +1034,7 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
 	  }
       }
       END_LOOP;
-      RestoreUndoSerialNumber ();
-      SelectConnection (Flag);
+      changed = SelectConnection (Flag);
       ResetFoundPinsViasAndPads (false);
       ResetFoundLinesAndPolygons (false);
       FreeConnectionLookupMemory ();
